pkg/drivemgr/drivemgr_service.go

```diff
@@ -66,3 +65,14 @@ func (svc *DriveServiceServerImpl) GetDrivesList(ctx context.Context, req *api.D
 		Disks: drives,
 	}, nil
 }
+
+// Locate invokes DriveManager's Locate method for manipulation drive's LED state
+func (svc *DriveServiceServerImpl) Locate(ctx context.Context, in *api.DriveLocateRequest) (*api.DriveLocateResponse, error) {
+	err := svc.mgr.Locate(in.GetDriveSerialNumber(), in.GetAction())
```

Comment:
I suggest to print waring when `codes.Unimplemented`

Reply:
I case of error we log message with ERROR level. Do you suggest to switch log level to WARNING if error codes is `unimplemented`? I think error level is suitable here.